uvtools/plot.py

```diff
@@ -40,241 +40,244 @@ def data_mode(data, mode='abs'):
 
 
-def plot_hmap_ortho(h, cmap='jet', mode='log', mx=None, drng=None, 
+def plot_hmap_ortho(h, cmap='jet', mode='log', mx=None, drng=None,
                     res=0.25, verbose=False, normalize=False):
     """
     Plot a Healpix map in ortho projection.
-    
+
     Parameters
     ----------
     h : aipy HealpixMap object
         HEALPIX map.
-    
+
     cmap : str, optional
         Which matplotlib colormap to use. Default: 'jet'.
-    
+
     mode : str, optional
-        Which transform to apply to the data before plotting. See the 
+        Which transform to apply to the data before plotting. See the
         `data_mode` function for available options. Default: 'log'.
-    
+
     vmin, vmax : float, optional
-        Minimum and maximum values of the color scale. If not set (and `mx` and 
+        Minimum and maximum values of the color scale. If not set (and `mx` and
         `drng` are not set), the min. and max. values of the data will be used.
-        
-        Note that that the min. and max. values are the ones _after_ the data 
-        have been transformed according to `mode`. So, if `mode='log'`, these 
+
+        Note that that the min. and max. values are the ones _after_ the data
+        have been transformed according to `mode`. So, if `mode='log'`, these
         values are the min. and max. of log_10(data).
-        
+
     mx : float, optional
-        The max. value of the color scale in the plot (equivalent to vmax). 
+        The max. value of the color scale in the plot (equivalent to vmax).
         Cannot be specified at the same time as `vmin` and `vmax`.
-    
+
     drng : float, optional
-        The difference between the min. and max. values of the color scale in 
-        the plot, `drng = mx - min`, where these are the min/max values after 
+        The difference between the min. and max. values of the color scale in
+        the plot, `drng = mx - min`, where these are the min/max values after
         applying the transformation specified by `mode`.
-        
+
         Cannot be specified at the same time as `vmin` and `vmax`.
-    
+
     res : float, optional
         Resolution of pixel grid, in degrees. Default: 0.25.
-    
+
     verbose : bool, optional
-        Whether to print basic debugging information. Default: False. 
-    
+        Whether to print basic debugging information. Default: False.
+
     normalize : bool, optional
-        Whether to normalize the data by the value at coordinates lat, long = 
+        Whether to normalize the data by the value at coordinates lat, long =
         (0, 0). Default: False.
-    
+
     Returns
     -------
     plot : matplotlib.imshow
         Healpix map in ortho projection.
     """
-    from mpl_toolkits.basemap import Basemap
-    
+    try:
+        from mpl_toolkits.basemap import Basemap
+    except:
+        print("plot_hmap_ortho requires Basemap. Try running 'pip install --user git+https://github.com/matplotlib/basemap.git'")
```

Comment:
This should `raise` on the line immediately afterwards.

Reply:
I've converted prints to `raise ModuleNotFoundError(<helpful installation suggestion>)`